torch/_utils.py

```diff
@@ -173,13 +173,7 @@ def _rebuild_sparse_tensor(layout, data):
     raise NotImplementedError("rebuilding sparse tensor for layout %s" % (layout))
 
 
-def _rebuild_xla_tensor(data, dtype, device, requires_grad):
-    tensor = torch.from_numpy(data).to(dtype=dtype, device=device)
-    tensor.requires_grad = requires_grad
-    return tensor
-
-
-def _rebuild_mlc_tensor(data, dtype, device, requires_grad):
+def _rebuild_device_tensor_from_numpy(data, dtype, device, requires_grad):
```

Comment:
code removal 🎉 

For BC reasons, could you add
```python
# Should not be used, only here to be able to load Tensors serialized with older versions of pytorch
_rebuild_xla_tensor = _rebuild_device_tensor_from_numpy
_rebuild_mlc_tensor = _rebuild_device_tensor_from_numpy
```
Python serialization saves the function name and location. So old checkpoint that rely on these functions need it to still exist to be loadable.